docs/04-operation-guides/operations/obsv-02-adjust-loki.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Adjust Loki log limits
+---
+
+By default, Loki stores up to 30 GB of data for a maximum of 5 days, with maximum ingestion rate of 3 MB/s. If the default size or time is exceeded, the oldest logs are removed first.
```

Comment:
Better write "If the default time is exceeded, ...". Logs are not deleted automatically if Loki runs out of space.

Reply:
same here, this is from SKR docs. Need to take a look at that too.